web/src/features/fireWeather/components/graphs/plotlyHelper.ts

```diff
@@ -95,24 +89,24 @@ export const populateGraphDataForTempAndRH = (
   tempPlumeColor?: string,
   rhPlumeColor?: string
 ) => {
```

Comment:
I'm seeing a warning (presumably from sonarQube) about "Missing return type on function" - could you address this please?

Reply:
It's also happening in a few other functions further down in the file

Reply:
Done!